build/js/live-editor.debugger.js

```diff
@@ -1,29 +1,44 @@
-var Poster = require("../node_modules/poster/lib/poster");
-var EventSim = require("../node_modules/eventsim/lib/eventsim");
-var basic = require("../node_modules/basic-ds/lib/basic");
+!function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{var f;"undefined"!=typeof window?f=window:"undefined"!=typeof global?f=global:"undefined"!=typeof self&&(f=self),f.iframeOverlay=e()}}(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+"use strict";
+
+/**
+ * Creates an overlay on top of an iframe that intercepts and retriggers mouse
+ * events.  The purpose of this is two-fold:
+ * - provide better user experience when a drag operation leaves iframe's bounds
+ * - allow events to be filtered to toggle interactivity without having to modify
+ *   the code runing inside the iframe
+ */
+
+var Poster = require("poster");
+var EventSim = require("eventsim");
+var LinkedList = require("basic-ds").LinkedList;
+
 function createOverlay(iframe) {
     var wrapper = document.createElement("span");
     wrapper.setAttribute("style", "position:relative;padding:0;margin:0;display:inline-block;");
     wrapper.setAttribute("class", "wrapper");
+
     var overlay = document.createElement("span");
     overlay.setAttribute("style", "position:absolute;left:0;top:0;width:100%;height:100%;");
     overlay.setAttribute("class", "overlay");
-    overlay.setAttribute("tabindex", "0");
+    overlay.setAttribute("tabindex", "0"); // allwos the span to have focus
+
     var parent = iframe.parentElement;
     parent.insertBefore(wrapper, iframe);
     wrapper.appendChild(iframe);
     wrapper.appendChild(overlay);
+
     var down = false;
     var paused = false;
-    var queue = new basic.LinkedList();
+    var queue = new LinkedList();
+
     var poster = new Poster(iframe.contentWindow);
+
     function postMouseEvent(e) {
         if (paused) {
-            e["timestamp"] = Date.now();
+            e.timestamp = Date.now(); // Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=238041
```

Comment:
I used [ ] originally b/c TypeScript doesn't like you adding properties if they aren't part of the type definition.